roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -42,22 +41,15 @@ class DockerStorage(DockerHostMixin, OpenShiftCheck):
     ]
 
     def run(self):
-        msg, failed = self.ensure_dependencies()
-        if failed:
-            return {
-                "failed": True,
-                "msg": "Some dependencies are required in order to query docker storage on host:\n" + msg
-            }
-
-        # attempt to get the docker info hash from the API
-        docker_info = self.execute_module("docker_info", {})
-        if docker_info.get("failed"):
-            return {"failed": True,
-                    "msg": "Failed to query Docker API. Is docker running on this host?"}
-        if not docker_info.get("info"):  # this would be very strange
-            return {"failed": True,
-                    "msg": "Docker API query missing info:\n{}".format(json.dumps(docker_info))}
-        docker_info = docker_info["info"]
+        # attempt to get the docker info hash from the docker cli
+        command = ' '.join(['docker', 'info', '--format', '"{{json .}}"'])
```

Comment:
wouldn't this fail when cri-o/ podman is installed ? or we saying that even with cri-o as container runtime, docker will be a must ?

Reply:
The `docker_storage` check is not run when `openshift_use_crio_only` is `True`.
https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/openshift_health_checker/action_plugins/openshift_health_check.py#L73-L75

The original `docker_info` module was just getting docker info through Ansible internals.  I'm slowing removing as much reliance on Ansible internals as possible because it is a continual problem as Ansible development continues.

Reply:
appreciate taking the time to respond @mtnbikenc ! keep up the excellent work 👍